development.ini

```diff
@@ -13,6 +13,9 @@ jujugui.sandbox = true
 jujugui.raw = true
 jujugui.combine = true
 
+# JEM connection
+jujugui.jem_url =
```

Comment:
Charmstore's url should also be here--it's a left over from the change to the wsgi form. If you look at jujugui/options.py you'll see we're also setting the charmstore url there.